addons/stock_landed_costs/tests/test_stock_landed_costs_lots.py

```diff
@@ -119,3 +119,61 @@ def test_stock_landed_costs_lots(self):
             {'lot_id': lot_product_b[0].id, 'product_id': product2.id, 'quantity': -2, 'value': -22.6},
             {'lot_id': lot_product_b[1].id, 'product_id': product2.id, 'quantity': -2, 'value': -22.6},
         ])
+
+    def test_landed_cost_when_partially_sold(self):
+        """
+        check that the landed costs split correctly between lot/ serial numbers
+        when some lot/serial number are empty (no share of the landed cost for those)
+        or when some have a portion of their quantity already sold (check that it uses the
+        remaining quantity)
+        """
+        product1 = self.env['product.product'].create({
+            'name': 'product2',
+            'is_storable': True,
+            'tracking': 'lot',
+            'lot_valuated': True,
+        })
+        self.product1.categ_id.property_cost_method = 'fifo'
```

Comment:
`categ_id` isn't required as of saas-18.1, so you _probably_ have to manually create one for the product